packages/fcl-wc/package.json

```diff
@@ -12,16 +12,6 @@
   "bugs": {
     "url": "https://github.com/onflow/fcl-js/issues"
   },
-  "jest": {
-    "transform": {
-      ".js": "jest-esm-transformer"
-    },
-    "testEnvironment": "./jest-test-environment.js",
-    "globals": {
```

Comment:
Do we still need this?

Reply:
It only needs to be present in the FCL and SDK packages.